memote/utils.py

```diff
@@ -280,10 +280,19 @@ def is_modified(path, commit):
     Returns
     -------
     bool
-        Whether or not the given path is among the modified files.
+        Whether or not the given path is among the modified files
+        and was not deleted in this commit.
 
     """
-    return path in commit.stats.files
+    try:
+        d = commit.stats.files["path"]
```

Comment:
I think this should be the `path` variable, not a string. Also, I prefer a more descriptive variable name. Maybe `changes`?

```suggestion
        changes = commit.stats.files[path]
```

Reply:
Sorry, I forgot to consider/comment on the proposed name change. Since the dictionary `d` is used three times in the same expression and self-explanatorily defined in the preceding line, I think this is clearer than having to break the code over another line.